boards/arm/rpi_pico/rpi_pico-pinctrl.dtsi

```diff
@@ -31,4 +31,24 @@
 			input-schmitt-enable;
 		};
 	};
+
+	spi0_default: spi0_default {
+		group1 {
+			pinmux = <SPI0_CSN_P17>, <SPI0_SCK_P18>, <SPI0_TX_P19>;
+		};
+		group2 {
+			pinmux = <SPI0_RX_P16>;
+			input-enable;
+		};
+	};
+
+	spi1_default: spi1_default {
```

Comment:
According to the [Pico Datasheet](https://datasheets.raspberrypi.com/pico/Pico-R3-A4-Pinout.pdf) the only "default" SPI is SPI0, I would say we shouldn't enable/default `spi1` defaults at all for the pico board.

Reply:
Rebasing and Removed spi1 and related definitions.

Question: i2c1 is also not set as default in this illustration. Is it have no problem?
(I made SPI1 by seeing that.)

Reply:
I didn't have a chance to review the i2c PR, but I would suggest we remove that for consistency in a separate PR.

Reply:
Thanks for reply. I understand and agree it.